### DISCUSSION TITLE
Question on updating a lookup field using the calculate function

### DISCUSSION THREAD

**Discussion by simplycurious** [Innofactor/XrmToolBox.BulkDataUpdater#151] [Q&A]
Hello, 

I've been using the bulk data updater as a Business Analyst to update records in Dynamics and the tool has been working great so far. I recently tried to update a lookup on inactive activity records using the tool and have run into an error : unable to cast object of type 'system.string' to type 'microsoft.xrm.sdk.entityreference'. The syntax im using to update the lookup (to the account entity) on the activity record is {regardingobjectid.new_customaccountid.accountid}. Can someone please help validate this syntax? I have not been able to find any documentation on updating lookup fields on the XRM tokens site. 

Thanks!

**Comment by rappen** [comment]
Hi,
Unfortunately, the **Calculate** only be possible to string/text column as target. If you have the latest version, it now isn't available for other types.

![image](https://user-images.githubusercontent.com/2572253/132095261-c2b27238-f2dd-4fcc-9f21-890b8a79e879.png)
_Disable for account-lookup._

I'm working on it to handle targets of other types 🤞

**Comment by rappen** [comment]
Sorry, I missed responding to this earlier:

It is now possible!
Make sure you get the guid id.
See my blog about the release in April: https://jonasr.app/bdu-1-2022-4/
